clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3569,6 +3557,69 @@ protected enum DelegateConsumerRecordResult {
     SKIPPED_MESSAGE
   }
 
+  private void recordNearlineLocalBrokerToReadyToServerLatency(
```

Comment:
Could you please wrap the entire function within a try-catch block, and tolerates all exception except InterruptException?

Reply:
Is this necessary ? The calling method already does this

Reply:
It was necessary; the previous diff was much more complicated; if we bubble the exception to the callers, we will end up with failing the ingestion.

The latest diff is fine, since we just added a new log.